lib/screens/employees/employee_list_view.dart

```diff
@@ -135,7 +135,7 @@ class _EmployeeListViewState extends State<EmployeeListView> {
                   maxRadius: 28.0,
                   foregroundImage: CachedNetworkImageProvider(employee?.avatarAttachmentUrl ?? ""),
                   child: Text(
-                    getInitials(employee?.name ?? "", 2),
+                    getInitials(employee?.name ?? "", 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.